data/config/ndef-qa.json

```diff
@@ -223,7 +223,7 @@
         "options": []
       },
       { 
-        "title": "Genus species",
+        "title": "Genus Species",
```

Comment:
So for this one the word species should intentionally be left lower case as this is a biological naming convention. For example , Homo sapiens.